middleware/jwt.go

```diff
@@ -159,17 +159,23 @@ func JWTWithConfig(config JWTConfig) echo.MiddlewareFunc {
 	}
 
 	// Initialize
-	parts := strings.Split(config.TokenLookup, ":")
-	extractor := jwtFromHeader(parts[1], config.AuthScheme)
-	switch parts[0] {
-	case "query":
-		extractor = jwtFromQuery(parts[1])
-	case "param":
-		extractor = jwtFromParam(parts[1])
-	case "cookie":
-		extractor = jwtFromCookie(parts[1])
-	case "form":
-		extractor = jwtFromForm(parts[1])
+	// Split sources
+	sources := strings.Split(config.TokenLookup, ",")
+	var extractor jwtExtractor
```

Comment:
If you want to extract from multiple places then your would need to have slice of extractor methods otherwise you are just using last method from `parts` slice.

NB: do not forget to update `config.TokenLookup` comment with this new feature

Reply:
I changed code to test every lookup that user configure. If user present 2 token lookup example of header and cookie,
the code will add header and cookie extractor to list and after that it will test extractors list. If one of them works, the token is set to it.